_dashboards/dashboards-assistant/alert-insight.md

```diff
@@ -0,0 +1,318 @@
+}
+```
+{% include copy-curl.html %}
+
+The following table describes the Assistant Summary API parameters.
+
+Parameter | Required/Optional | Description
+:--- | :--- | :---
+`summaryType` | Required | Specifies the type of application calling this API. Use `alerts` for alert insights.
+`question` | Required | Specifies the user's question regarding alert insights. Default is `Please summarize this alert, do not use any tool.` 
```

Comment:
I realize this may not be changeable, but the default should read "Please summarize this alert. Do not use any tool."